shell/common/platform_util_linux.cc

```diff
@@ -13,13 +11,32 @@
 #include "base/process/kill.h"
 #include "base/process/launch.h"
 #include "chrome/browser/ui/libgtkui/gtk_util.h"
+#include "shell/common/platform_util.h"
```

Comment:
nit: this shouldn't be moved, https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes